ssz/sedes/byte_vector.py

```diff
@@ -27,6 +28,9 @@ def get_fixed_size(self):
     #
     # Serialization
     #
+    def get_element_sedes(self, index: int) -> BaseSedes:
+        raise NotImplementedError()
```

Comment:
It might be nice to have a comment in these cases explaining why this has to be here...

Reply:
This will be just temporary. The problem is that bitlists/bitvectors/bytevectors are all technically composite SSZ objects, but don't really have elements that can (or should) be independently serialized. The current HashableStructure just distinguishes between composites and basics for computing the tree root, so it doesn't work for those cases. I still have to implement the method though as it's an abstract method in `BaseCompositeSedes`. I've added a `TODO` comment for now.